force-app/main/default/lwc/mortgage/__tests__/mortgage.test.js

```diff
@@ -0,0 +1,39 @@
+import { getTermOptions, calculateMonthlyPayment } from 'c/mortgage';
+
+describe('c-mortgage', () => {
+    describe('calculateMonthlyPayment function', () => {
+        it('returns monthly payment with valid inputs', () => {
+            const principal = 50000;
+            const years = 20;
+            const rate = 3;
+
+            // Get a two decimal place string representation of long decimal
+            const expectedResult = (277.298798926959).toFixed(2);
+
+            const monthlyPayment = calculateMonthlyPayment(
+                principal,
+                years,
+                rate
+            );
+            // convert long decimal to two decimal places and compare
+            expect(monthlyPayment.toFixed(2)).toBe(expectedResult);
+        });
+        it('returns zero with invalid inputs', () => {
+            // each value tests each param as falsy, third param also tested for explicit zero value
+            const invalidInputs = require('./data/invalidInputsTestData.json');
+
+            // invoke with 4 invalid input sets and store each return value in an array
+            const results = invalidInputs.inputs.map((item) =>
+                calculateMonthlyPayment(item.principal, item.years, item.rate)
+            );
+
+            expect(results).toMatchObject(invalidInputs.expectedResults);
+        });
+    });
+
+    it('getTermOptions returns valid term options', () => {
```

Comment:
@muenzpraeger I'd like your advice on this one.

I don't think that testing this makes sense because we're duplicating constant data in our test data. There's no particular logic under test here.
IMHO the `mortage` implementation should be changed to just export a constant instead of a function:

```js
const TERM_OPTIONS = [ ... ];
...
export { TERM_OPTIONS, calculateMonthlyPayment };
```

Reply:
I agree, actually. This test is pointless. And the actual use of it in `c/miscSharedJavaScript` should also be implemented as a getter (instead of a property pass through). 

I'll add this to the work expected in issue #323. 

I can leave this for now, and refactor later when I come back.

Reply:
My initial thinking was to agree with you. 

But is this intending to mock what would often be a dynamic set of picklist values? If so, then I'd argue with keeping the test case.

Reply:
At least we should test the default values. Agreeing with @pchittum.